### ISSUE TITLE
AttributeError: module 'tensorflow' has no attribute 'estimator'

### ISSUE THREAD

**Issue by mkuangdotcom** [tensorflow/tensorflow#76349]
### Issue type

Support

### Have you reproduced the bug with TensorFlow Nightly?

Yes

### Source

binary

### TensorFlow version

2.17

### Custom code

Yes

### OS platform and distribution

MacOS

### Python version

3.9.6

### Current behavior?

AttributeError: module 'tensorflow' has no attribute 'estimator'

### Standalone code to reproduce the issue

```shell
Cell In[15], line 2
      1 # Build the model
----> 2 model = tf.estimator.DNNClassifier(
      3     feature_columns=feature_columns,
      4     hidden_units=[64, 32],
      5     n_classes=2,
      6     model_dir='./model_dir')
      8 # Train the model
      9 model.train(input_fn=train_input_fn, steps=5000)

AttributeError: module 'tensorflow' has no attribute 'estimator'
```

**Comment by Venkat6871** [comment]
Hi **@mkuangdotcom** ,
Thank you for raising the issue here. You are using a deprecated API, which is clearly mentioned in the documentation. Estimators will not be available in TensorFlow 2.16 or later. I am providing the [documentation](https://www.tensorflow.org/guide/estimator) for your reference.
Thank you!

**Comment by mkuangdotcom** [comment]
Thanks for the info!
